server/plugin/template.go

```diff
@@ -350,7 +350,57 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 		"  * `/github mute list` - list your muted GitHub users\n" +
 		"  * `/github mute add [username]` - add a GitHub user to your muted list\n" +
 		"  * `/github mute delete [username]` - remove a GitHub user from your muted list\n" +
-		"  * `/github mute delete-all` - unmute all GitHub users\n"))
+		"  * `/github mute delete-all` - unmute all GitHub users\n" +
+		"* `/github Webhook` - Managed to List Webhook of [owner/repo] and add webhook to owoner[/repo].\n" +
+		"  * `/github Webhook list` - list your Webhook of a GitHub repository\n" +
+		"  * `/github webhook add [config] \n" +
+		"  * `config` is a comma-delimited list of 'url:,token:,insecure_ssl:,content_type:'\n" +
+		"    * `url` - Optional - Callback Url , if not passed considered as default `SITE_URL` + `/plugins/github/webhook` \n" +
+		"    * `token` - Optional - webhook secret\n" +
+		"    * `content_type` - Optional - application/json or application/x-www-form-encoded \n" +
+		"    * `insecure_ssl` - Optional - true or false \n" +
+		"  	 * `options` is a comma-delimated can have one or more values `options:value1&value2`\n" +
```

Comment:
Why is a `value` needed? Could these just be enable flags?

Reply:
yes , will make it with slash command